docs/sgp40/raspisetup.rst

```diff
@@ -0,0 +1,96 @@
+SGP40 on Raspberry Pi
+=====================
+
+This document explains how to set the SGP40 up to run with a Raspberry
+Pi. This became very easy with the new SGP40 since it can run with a
+logic level of 3.3v which matches the one the Raspberry Pi has.
+
+Getting started
+---------------
+
+Connecting the Sensor
+~~~~~~~~~~~~~~~~~~~~~
+
+| Your Sensor should have four different connectors: VCC, GND, SDA, SCL.
+Please refere to other documentation of the Sensor to find out which
+connector is which on your specific modelle.
+| Use the following pins to connect your SGP40:
+
++-----------+------------------+
+| *SGP40*   | *Raspberry Pi*   |
++===========+==================+
+| VCC       | Pin 1            |
++-----------+------------------+
+| GND       | Pin 6            |
++-----------+------------------+
+| SDA       | Pin 3            |
++-----------+------------------+
+| SCL       | Pin 5            |
++-----------+------------------+
+
+|RaspberryPi Pinout|
+
+Raspberry Pi
+~~~~~~~~~~~~
+
+-  **Install the newest Raspberry Pi OS on to your Raspberry Pi. If you
+   don't know how to do that, there is a guide on the Raspberry Pi
+   website**
```

Comment:
The bold is because the formatting automatically makes bullet points with sub points bold but those without them not. So to keep the formatting consistent I made them bold manually.